compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx

```diff
@@ -42,348 +16,354 @@ You can create and manage your Bare Metal servers from the console, via the API
+      IPs.0.ReverseStatusMessage  -
+      IPs.1.ID                    12345678-1234-1234-1234-123456789123
+      IPs.1.Address               1234:123:4567:8:9123:4567:8912:345
+      IPs.1.Reverse               12345678-1234-1234-1234-123456789123.fr-par-2.baremetal.scw.cloud
+      IPs.1.Version               IPv6
+      IPs.1.ReverseStatus         active
+      IPs.1.ReverseStatusMessage  -
+      Domain                      12345678-1234-1234-1234-123456789123.fr-par-2.baremetal.scw.cloud
+      BootType                    normal
+      Zone                        fr-par-2
+      Install.OsID                ID-of-your-OS
+      Install.Hostname            hostname-for-your-server
+      Install.SSHKeyIDs.0         your-ssh-key-ID
+      Install.Status              to_install
+      PingStatus                  up
+      ```
+
+<Message type="note">
+
+You can check the status of your server with the command `scw baremetal server list zone=fr-par-2`.
+
+</Message>
 
-  Installation of the OS should take less than one hour. You can use the `scw baremetal server list zone=fr-par-2` command again if you wish, to check whether the status of your server has moved from `Installing` to `Ready`.
+
+
+
+ 
 
 ## Using other CLI commands
 
-There are many other functionalities you can access for your Bare Metal Server via the Scaleway CLI. In the following examples, replace the Bare Metal Server ID with the ID of your Bare Metal server:
+There are many other functionalities you can access for your Bare Metal server via the Scaleway CLI. In the following examples, replace `your-bare-metal-server-ID` with your server's ID. 
 
 <Message type="note">
 
-Before powering off your Bare Metal server, you should [log into your server as root](/compute/bare-metal-cloud/how-to/connect-to-server) and execute the `halt` command to shutdown the OS. If you're not sure of the IP address of your Bare Metal server, for login purposes, use the `scw baremetal server get <Server-ID/> zone=fr-par-2` command to see this information in the `IPs.0.Address` field
+ * Before powering off your server, we recommend you [log into your server as root](https://www.scaleway.com/en/docs/compute/bare-metal-cloud/how-to/connect-to-server/) and execute the `halt` command to shutdown the OS. 
+ * Type `scw baremetal server get name-of-server-ID zone=fr-par-2` in your terminal if you are not sure what the IP address of your Bare Metal server is. It will display in the `IPs.0.Address` field. 
 
 </Message>
 
--**To power off** your Bare Metal Server, use the following command:
+### Powering on your Server
 
-  ```
-  scw baremetal server stop <Bare Metal Server ID> zone=fr-par-2
-  ```
+Type the following command in your terminal:
 
--**To power on** your Bare Metal Server, use the following command:
+```
+scw baremetal server start your-bare-metal-server-ID zone=fr-par-2
+```
 
-  ```
-  scw baremetal server start <Bare Metal Server ID> zone=fr-par-2
-  ```
+<Message type="note">
```

Comment:
I think the last sections looks a bit message-box-heavy. What do you think about removing the message boxes around the bits that show the result of each command?